Pipfile

```diff
@@ -48,7 +48,7 @@ python_version = "3.7"
 allow_prereleases = true
 
 [scripts]
-serve = "flask run -p ${FLASK_SERVE_PORT}"
+serve = "flask run --eager-loading -p ${FLASK_SERVE_PORT}"
```

Comment:
This change is made to ensure that running the dev server locally fails immediately if there are unapplied migrations. Normally, this runs in a "lazy loading" mode, which won't throw this error until a request has been made.